### PR TITLE
feature: HU15

### PR DESCRIPTION
**Pull request by alejahenaoes**

### CODE REVIEW

File: html/aboutUs.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                <img src="https://tuwebsoluciones.com/wp-content/uploads/2018/03/team-member-4.jpg" alt="team-member" class="photo-member6 member-photo">
+                <div class="member-info">
+                    <p class="name-member6 name-member">Óscar</p>
+                    <p class="position-member6">Maquetador</p>
+                    <div class="social-media">
+                        <a href="" class="twitter tw-member6"><img src="https://cdn4.iconfinder.com/data/icons/social-media-icons-the-circle-set/48/twitter_circle-512.png" alt=""></a>
+                        <a href="" class="linkedin ln-member6"><img src="https://www.oseyo.co.uk/wp-content/uploads/2020/04/linkedin-icon-png-circle-2.png" alt=""></a>
+                    </div>
+                </div>
+            </div> -->
```
Comment (Salacodigo) [inline html/aboutUs.html:152]: Hi Aleja! the commented block will be included or not?

File: styles/scss/_aboutValues.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                font-size: 48px;
+            }
+            .title-counter{
+                font-size: 23px;
+            }
+        }
+    }
+    .values{
+        display: flex;
+        // margin-right: 100px;
```
Comment (Salacodigo) [inline styles/scss/_aboutValues.scss:76]: @alejahenaoes take a look at this commented line

File: styles/scss/_aboutValues.scss [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                        font-weight: normal;
+                        font-size: 20px;
+                        margin: 35px auto 15px ;
+                    }
+            }
+        }
+        .our-ADN{
+            .ourValues{
+                // width: 500px;
+                // height: fit-content;
```
Comment (Salacodigo) [inline styles/scss/_aboutValues.scss:106]: @alejahenaoes Take a look at this commented lines

**Review by Salacodigo — APPROVED** [review]
LGTM

File: html/aboutUs.html (hunk 1 above)
Comment (alejahenaoes) [reply html/aboutUs.html:152]: @Salacodigo it's possible that we need it

**Pull request merged**